Frontend/src/Components/Temperature/TempView.js

```diff
@@ -0,0 +1,70 @@
+import {Flex, VStack, Box} from "@chakra-ui/react";
+import RangeRow from "../Shared/RangeRow";
+import CONSTANTS from "../../data-constants.json";
+
+
+export default function TempView(props) {
+
+    console.log(props.data);
+    console.log(CONSTANTS)
+    return (
+        <Flex flex="auto" direction="column" overflowY="scroll">
+            <RangeRow
+                dataConstant={CONSTANTS.motor_temp}
+                dataValue={props.data.motor_temp[0] ?? CONSTANTS.motor_temp.MIN}
```

Comment:
```suggestion
                dataValue={props.data.motor_temp[0] ?? -1}
```
For reference when this feature is implemented again: Typically, we use -1 (or false/whatever will make the component red/hidden/more obviously null) to indicate that the signal isn't being received.